docs/src/components/IconsList.tsx

```diff
@@ -0,0 +1,151 @@
+    width: 20px;
+    visibility: hidden;
+  }
+
+  &:hover svg {
+    visibility: visible;
+  }
+`;
+
+interface IIconObject {
+  character: string;
+  description: string;
+  svg: string;
+  url: string;
+}
+
+interface IconProps {
+  iconName: string;
+  opened: boolean;
+  setOpenedIcon: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const Icon = ({ iconName, opened, setOpenedIcon }: IconProps) => {
+  const [copied, setCopied] = useState(false);
+  useEffect(() => {
+    copyTimeout(copied, setCopied);
+  }, [copied, setCopied]);
+
+  const iconObject: IIconObject = iconsList[iconName];
+  const pascalName = pascalize(iconName);
+
+  return (
+    <div id={iconName}>
+      <Tile onClick={() => setOpenedIcon(opened ? "" : iconName)}>
+        <Stack justify="between">
+          <img src={`data:image/svg+xml;utf8,${iconObject.svg}`} alt="" />
+          <Coupon>{iconObject.character}</Coupon>
+        </Stack>
+        <Text type="secondary">{iconName}</Text>
+      </Tile>
+      {opened && (
+        <Card>
+          <CardSection>
+            <Stack>
+              <StyledAnchorWrapper>
+                <Heading as="h3" type="title2" spaceAfter="medium">
+                  <a href={`#${iconName}`}>
+                    <LinkIcon />
+                  </a>
+                  {iconName}
+                </Heading>
+              </StyledAnchorWrapper>
+              {iconObject.description && <Text>{iconObject.description}</Text>}
+              <CodeBlock className="language-jsx">{`import { ${pascalName} } from "@kiwicom/orbit-components/icons"\n\n<${pascalName} />`}</CodeBlock>
+              <Stack direction="row" justify="between">
+                <Button size="small" type="secondary" href={iconObject.url} external>
+                  Download SVG
+                </Button>
+                <CopyToClipboard text={iconObject.svg}>
+                  <Button
+                    size="small"
+                    type="secondary"
+                    disabled={copied}
+                    onClick={() => setCopied(true)}
+                  >
+                    {copied ? "Copied source!" : "Copy SVG source"}
+                  </Button>
+                </CopyToClipboard>
+              </Stack>
+            </Stack>
+          </CardSection>
+        </Card>
+      )}
+    </div>
+  );
+};
+
+const IconsList = () => {
+  const allIconNames = Object.keys(iconsList);
+  const [openedIcon, setOpenedIcon] = useState("");
+  const [filter, setFilter] = useState("");
+
+  useEffect(() => {
+    if (typeof window !== "undefined") setOpenedIcon(window.location.hash.replace("#", ""));
+  }, []);
+
+  const filteredIconNames = allIconNames.filter(name => name.includes(filter.toLowerCase()));
+
+  return (
+    <Stack spacing="large">
+      <InputField
+        prefix={<Search />}
+        placeholder="Find an icon"
+        value={filter}
+        onChange={event => {
+          if (event.target instanceof HTMLInputElement) {
```

Comment:
you can remove that condition and rewrite it like `onChange={ev => setFilter(ev.currentTarget.value)}`